README.asciidoc

```diff
@@ -1,2 +1,7 @@
-TODO
-Instructions how to set up and run the code.
```

Comment:
A short description as to what this code does would also be good.

Reply:
done